bolt-modules/boltlib/lib/puppet/functions/set_var.rb

```diff
@@ -28,6 +28,12 @@ def set_var(target, key, value)
     executor = Puppet.lookup(:bolt_executor)
     executor.report_function_call(self.class.name)
 
-    inventory.set_var(target, key, value)
+    # Inventory v2 expects a hash, should we just update v1 as well?
```

Comment:
I don't see a reason not to :woman_shrugging:

Reply:
updated